client/src/components/Toasts/Toasts.js

```diff
@@ -0,0 +1,39 @@
+import React, {useEffect, useState} from 'react';
+import PropTypes from 'prop-types';
+import i18n from 'i18n';
+import { Toast as BsToast, ToastBody, ToastHeader } from 'reactstrap';
+import classnames from 'classnames';
+
+const Toast = ({type, text, onDismiss, dismissed}) => {
+  const toggle = (e) => {
+    e.preventDefault();
+    onDismiss();
+  };
+
+  const [show, setShow] = useState(false);
+  useEffect(() => {
+    setShow(!dismissed);
+  }, [dismissed]);
+
+  const className = classnames('toast', `toast--${type}`, {'toast--dismissing': dismissed});
+
+  return (
+    <BsToast className={className} isOpen={show} aria-live="assertive" aria-atomic="true">
```

Comment:
The aria attributes in this file comes from the [accessibility suggestion on the bootstrap site](https://getbootstrap.com/docs/4.3/components/toasts/#accessibility).